packages/react-native-gesture-handler/src/web/tools/KeyboardEventManager.ts

```diff
@@ -3,25 +3,49 @@ import EventManager from './EventManager';
 import { PointerType } from '../../PointerType';
 
 export default class KeyboardEventManager extends EventManager<HTMLElement> {
-  private activationKeys = ['Enter', ' '];
-  private cancelationKeys = ['Tab'];
+  private static activationKeys = ['Enter', ' '];
+  private static cancelationKeys = ['Tab'];
   private isPressed = false;
+  private static registeredStaticListeners = false;
+  private static instances: Set<KeyboardEventManager> = new Set();
+
+  private static keyUpStaticCallback = (event: KeyboardEvent): void => {
+    /*
+     We need a global listener, as in some cases, keyUp event gets stop-propagated.
+     Then, if we used only component-level listeners the gesture would never end,
+     causing other gestues to fail.
+     */
```

Comment:
Just one mor thing, can we stick to one convention of writing comments? 😅 We tend to use `//` instead of multiline comments (e.g. [here](https://github.com/software-mansion/react-native-gesture-handler/blob/2d19b817ea60e822a73161a29f5e8f26dd3014ae/packages/react-native-gesture-handler/src/web/tools/PointerEventManager.ts#L88))